reconciletags/reconciletags_test.py

```diff
@@ -110,6 +110,22 @@ def test_add_tag(self, mock_from_registry, mock_push):
         assert mock_from_registry.called
         assert mock_push.called
 
+    @patch('containerregistry.client.v2_2.docker_image.FromRegistry')
+    def test_get_digest(self, mock_from_registry):
+        fake_base = mock.MagicMock()
+        fake_base.manifests.return_value = ["sha256:" + _DIGEST1,
+                                            "sha256" + _DIGEST2]
+
+        mock_img = mock.MagicMock()
+        mock_img.__enter__.return_value = fake_base
+        mock_from_registry.return_value = mock_img
+
+        digest = self.r.get_digest(_FULL_REPO, "0")
```

Comment:
Is "0" the first character in one of the _DIGEST constants? Maybe use it that way instead? _DIGEST[0:3] or something to be more clear it's a prefix.

A test case involving multiple matches would be good too.

Reply:
yah, 0 is the first character, i'll make it clearer and add the test case!